.github/workflows/build-docker.yml

```diff
@@ -35,7 +35,7 @@ jobs:
             buildtype: "reldeb"
             image: "rocky-cuda"
     env:
-      CELER_TEST_STRICT: 1
+      CELER_TEST_STRICT: 0 # TODO: enable with Geant4 v11.3
```

Comment:
Is it ok to disable strict testing for the docker build, which uses 11.0.4?

Reply:
Yep! Since we're not actually running the CUDA.